bin/node/cli/src/service.rs

```diff
@@ -320,7 +321,11 @@ pub fn new_full_base(
 	let hwbench = if !disable_hardware_benchmarks {
 		config.database.path().map(|database_path| {
 			let _ = std::fs::create_dir_all(&database_path);
-			sc_sysinfo::gather_hwbench(Some(database_path))
+			sc_sysinfo::gather_hwbench(
+				Some(database_path),
+				SUBSTRATE_REFERENCE_HARDWARE.clone(),
```

Comment:
Its a bit unfortunate that we have to pass this in twice now; once for the sysinfo and once for the machine benchmarking.  
But I dont see how it could be done better.